openapi.yaml

```
@@ -480,6 +481,35 @@ paths:
 
                   result = client.apps.runs.create(app_name='<APP-NAME>',
                                                    batch_id='<BATCH-ID>')
+        - code-samples:
+            - sdk: curl
+              name: Request with post-processed-pdf generation enabled
```

```suggestion
              name: Post-processed-pdf generation enabled
```

Shortening to help with narrow browser windows.